src/Storages/FileLog/StorageFileLog.cpp

```diff
@@ -175,7 +175,7 @@ StorageFileLog::StorageFileLog(
     , path(path_)
     , metadata_base_path(std::filesystem::path(metadata_base_path_) / "metadata")
     , format_name(format_name_)
-    , log(getLogger("StorageFileLog (" + table_id_.table_name + ")"))
+    , log(getLogger("StorageFileLog (" + table_id_.getNameForLogs() + ")"))
```

Comment:
Sorry, I did not realize `getNameForLogs()` prints UUID and I don't really like how it looks there. What do you think about using this instead?
```suggestion
    , log(getLogger("StorageFileLog (" + table_id_.getFullTableName() + ")"))
```

Reply:
Yeah, I prefer `getFullTableName()`, `getNameForLogs()` with UUID is so lengthy.

`getNameForLogs()` log will be like:
```
2025.02.06 08:31:31.978022 [ 34696 ] {} <Warning> StorageKafka (default.eco_service_flow_realtime_kafka(028cd529-8e11-4802-9eac-a49da9c737c6)): Can't get assignment. Will keep trying.
```
`getFullTableName()` log will be like:
```
2025.02.06 08:31:31.978022 [ 34696 ] {} <Warning> StorageKafka (default.eco_service_flow_realtime_kafka): Can't get assignment. Will keep trying.
```